### ISSUE TITLE
test

### ISSUE THREAD

**Issue by xuedinge233** [xuedinge233/triton-ascend#61]
ascendnpu-ir_1.2.0_linux-x86.run
6230709fb43f75dd6247128163fd417683429ac86852f972f1a5724a19dc2225

**Comment by xuedinge233** [comment]
import torch, torch_npu, triton, sys
torch.manual_seed(0)
torch_npu.npu.utils.set_device(0)
sys.path.insert(0, 'third_party/ascend/unittest/pytest_ut')
import test_common, test_elementwise_round as t

x = test_common.generate_tensor(shape=(128,), dtype='float16')   # [-256-256] 用例的输入
ans = t.standard_round(x)
out = torch.zeros(128, dtype=torch.float16).npu()
t.triton_round[1, 1, 1](x.npu(), out, N=128, NUMEL=256, debug=True)

print("x[75]   =", x[75].item())
print("ref[75] =", ans[75].item())
print("out[75] =", out[75].cpu().item())

**Comment by xuedinge233** [comment]
(ta-3.6) root@triton:/data/c00911783/workspace/triton-ascend# python
Python 3.11.15 (main, Jun 11 2026, 15:20:16) [GCC 14.3.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import torch, torch_npu, triton, sys
seed(0)
torch_npu.npu.utils.set_device(0)
sys.path.insert(0, 'third_party/ascend/unittest/pytest_ut')
import test_common, test_elementwise_round as t

x = test_common.generate_tensor(shape=(128,), dtype='float16') # [-256-256] 用例的输入
ans = t.standard_round(x)
out = torch.zeros(128, dtype=torch.float16).npu()
t.triton_round[1, 1, 1](x.npu(), out, N=128, NUMEL=256, debug=True)

print("x[75] =", x[75].item())
print("ref[75] =", ans[75].item())
print("out[75] =", out[75].cpu().item())/home/c00911783/miniconda3/envs/ta-3.6/lib/python3.11/site-packages/torch_npu/_init/core/optional_features.py:20: UserWarning: On the interactive interface, the value of TASK_QUEUE_ENABLE is set to 0 by default. Do not set it to 1 to prevent some unknown errors
  warnings.warn(
>>> torch.manual_seed(0)
<torch._C.Generator object at 0x7f0a363eceb0>
>>> torch_npu.npu.utils.set_device(0)
^[[C
>>> sys.path.insert(0, 'third_party/ascend/unittest/pytest_ut')
>>> import test_common, test_elementwise_round as t
>>> 
>>> x = test_common.generate_tensor(shape=(128,), dtype='float16') # [-256-256] 用例的输入
>>> ans = t.standard_round(x)
>>> out = torch.zeros(128, dtype=torch.float16).npu()
>>> t.triton_round[1, 1, 1](x.npu(), out, N=128, NUMEL=256, debug=True)
Dumping precompiled.h to /root/.triton/dump/FSD72WAEHDK2XYLPF6DCJ772LEBLTATVR7YO623K74IH2DT7IUQA
Dumping launcher_cxx11abi1.cxx to /root/.triton/dump/FSD72WAEHDK2XYLPF6DCJ772LEBLTATVR7YO623K74IH2DT7IUQA
<triton.compiler.compiler.CompiledKernel object at 0x7f09fb651ad0>
>>> 
>>> print("x[75] =", x[75].item())
x[75] = 0.2666015625
>>> print("ref[75] =", ans[75].item())
ref[75] = 0.0
>>> print("out[75] =", out[75].cpu().item())
out[75] = 0.0
